index.html

```diff
@@ -60,219 +65,122 @@
           </nav>
           <section class="container headline-section d-flex flex-column">
     
-                <h2 class="text-color fw-light">"Hello! Sharing World"</h2>
-                <h1 class="text-color"> Creative Commoons Global Summit 2015</h1>
+                <span class="text-color-orange fw-lighter headline-span">"Hello! Sharing World"</span>
+                <h1 class="text-color-orange fw-bolder headline-h1"> Blockchain For Everyone: Learn without Limits</h1>
                 <div class="card my-4">
                     <div class="card-body  bg-container-light border border-2 border-white">
-                    <p class="card-text">Some quick example text to build on the card
-                     title and make up the bulk of the card's content.</p>
+                    <p class="card-text text-color-dark"><small> You can start with advance or beginner level with the best lecturers and platform. Advance your
+                        career with us, be part of the future.
+                    </small></p>
                     </div>
                 </div>
-                <h2>2022.10.15(THU) ~ 16(FRI)</h2>
-                <p>National Musuem of Korea, Art center Nabi and more</p>
+                <span class="headline-span text-color-dark fw-bolder lh-lg">2022.10.15(THU) ~ 16(FRI)</span>
+                <p class="text-color-dark ">@ On our platform online , register to join.</p>
             
         </section>
     </header>
     <main class="container-fluid px-0">
-        <section class="container-fluid text-center bg-container py-5">
-            <h2 class="text-light">Main Program</h2>
+        <section class="container-fluid text-center bg-container-dark py-5" id="program-section">
```

Comment:
Kindly update this section with a background image as in the provided design. Also, consider making cards of the same height. I am sure it would look much better if all the cards were of the same height.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/192083873-2b06b57a-1ca3-444f-92c2-211f55f62363.png) | ![image](https://user-images.githubusercontent.com/33268927/192083886-4b72c573-0bad-4dcd-8fc5-5402c838cc57.png) |

Reply:
- [ ] @yetemegn-telaye as previously requested kindly ensure this section has a background image and the program cards have equal height, it currently seems their height is dynamically adjusted depending on the content size you can change it to a fixed height. You can see below what they look like:

![image](https://user-images.githubusercontent.com/61408860/192111066-2c881565-5c3a-467d-871b-e3b85483939a.png)